Update apolloClient query .await()

AndreSand opened the pull request:
Hi, I was following the Apollo tutorial and I had to add `toDeferred()` to get it to work:
from:
val response = apolloClient.query(LaunchListQuery()).await()

to:
val response = apolloClient.query(LaunchListQuery()).toDeferred().await()

apollo-cla commented:
@AndreSand: Thank you for submitting a pull request! Before we can merge it, you'll need to sign the Apollo Contributor License Agreement here: https://contribute.apollographql.com/

martinbonnin commented:
Hi @AndreSand 👋 ! Thanks for reaching out! There should be a `fun ApolloCall.await()` (see https://github.com/apollographql/apollo-android/blob/main/apollo-coroutines-support/src/main/kotlin/com/apollographql/apollo/coroutines/CoroutinesExtensions.kt#L90). 

Can you double check your imports? Maybe add something like this?

```
import com.apollographql.apollo.coroutines.await
```

AndreSand commented:
Hi @martinbonnin, I added the import you mentioned above, but it didn't worked.

I updated to latest apollo version `2.5.3` and it worked using:
`val response = apolloClient.query(LaunchListQuery()).await()`

Maybe we can update the documentation to use apollo latest version 2.5.3.
